src/components/pages/Chat/EmptyFeed/index.tsx

```diff
@@ -13,16 +15,34 @@ const CreatedAt = styled.span`
     margin-bottom: 2rem;
 `
 
-const UnstyledEmptyFeed = ({ className, roomCreatedAt }: Props) => {
+const UnstyledEmptyFeed = ({ className }: Props) => {
+    const [roomCreatedAt, setRoomCreatedAt] = useState(0)
+
+    const {
+        roomId,
+        session: { streamrClient },
+    } = useStore()
+
+    useEffect(() => {
+        const fn = async () => {
+            if (!streamrClient || !roomId || roomCreatedAt !== 0) {
+                return
+            }
+            const stream = await streamrClient.getStream(roomId)
+            const description = getRoomDescription(stream)
```

Comment:
At this point we (this component) may not be mounted anymore. Let's use the pattern

```js
useEffect(() => {
    let mounted = true

    (async () => {
        await Promise.resolve() // Await something.
        if (!mounted) {
            return
        }
        // Post-await code while we know we mounted.
    })()

    return () => {
        mounted = false
    }
}, [/* …deps */])
```